remove_comments.py

```diff
@@ -0,0 +1,17 @@
+"""
+Silly little module which removes comments.
+
+This is just so we can check what happens when running nbqa on a tool which causes a failure.
+"""
+
+import argparse
```

Comment:
Should this module be placed under the tests directory, since this is only used for testing purposes and does not serve any use to the end user. We can add `tests/remove_comments.py` to PYTHONPATH before launching the tests and clean up after the test runs?

In `__main__.py`, may be we need to change `env["PYTHONPATH"] = os.getcwd()` to `env["PYTHONPATH"] = f"{env["PYTHONPATH"]}{os.pathsep}{os.getcwd()}`

Reply:
Do you see any issues if we append to existing values of `PYTHONPATH`?

Reply:
That's a good point, best to not clutter the root folder too much. I've made a fixture which makes a temporary copy.

> In __main__.py, may be we need to change env["PYTHONPATH"] = os.getcwd() to env["PYTHONPATH"] = f"{env["PYTHONPATH"]}{os.pathsep}{os.getcwd()}

Yes, this seems like a good idea...I'll make a separate PR for that

Reply:
Ah! I like the fixture idea!